src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentTypeFinder.cs

```diff
@@ -390,7 +384,7 @@ internal static class DependentTypeFinder
                                                  .ToList();
         }
 
-        private static async Task<IEnumerable<INamedTypeSymbol>> FindMetadataTypesInProjectAsync(
+        private static async Task<IEnumerable<INamedTypeSymbol>> FindAllMatchingMetadataTypesInProjectAsync(
             HashSet<INamedTypeSymbol> metadataTypes,
```

Comment:
What does `metadataTypes` represent? Some sort of starting set that the function builds off of?